conda/meta.yaml

```diff
@@ -18,24 +18,24 @@ requirements:
     - pip
     - pytest-runner
     - python >=3.6,<3.9
-    - scikit-learn >=0.23,<1
+    - scikit-learn >=0.24,<2
```

Comment:
Why is the new upper bound so high? I think for libraries lower than 1.0.0, we usually set the upper limit to one middle value higher. For example, here it would be `<0.3`.

Reply:
The newest version is `1.0.1` already - https://pypi.org/project/scikit-learn/#history

I don't see any reason to not take the latest version. We could do `<1.1` if that's preferred.

Reply:
my understanding is that when the major version is `0`, sometimes the next update could be API breaking even if it only updates the minor version

Reply:
I see. But in this case we've already verified that it can use up to `1.0.1`.

Reply:
I agree with Katharine here: Now`scikit-learn` is already passed the 1.0 version, so we can expect them to not break the API until they advance to 2.x.